install_config/router/default_haproxy_router.adoc

```diff
@@ -846,6 +846,8 @@ $ oc env dc/router ROUTER_ALLOW_WILDCARD_ROUTES=true
 $ oc scale dc/router --replicas=1
 ----
 
+xref:../../install_config/web_console_customization.adoc#web-console-enable-wildcard-routes[Learn how to configure the web console for wildcard routes].
```

Comment:
Is this consistent with how we normally link to other doc pages, seems like a lot of link text
cc @ahardin-rh ?